### PR TITLE
Added ProxyConfigHolder class. Added StepTest class.

### PR DESCRIPTION
**Pull request by MBMIII**

### CODE REVIEW

File: src/test/java/executor/service/StepTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
+package executor.service;
+
+import executor.service.model.Step;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StepTest {
+
+    String action = "click";
```
Comment (artemkedyk) [inline src/test/java/executor/service/StepTest.java:9]:
```suggestion
    private final String action = "click";
```

File: src/test/java/executor/service/StepTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    String value = "button";
```
Comment (artemkedyk) [inline src/test/java/executor/service/StepTest.java:10]:
```suggestion
    private final String value = "button";
```

File: src/test/java/executor/service/StepTest.java (hunk 1 above)
Comment (MBMIII) [reply src/test/java/executor/service/StepTest.java]: updated

File: src/test/java/executor/service/StepTest.java (hunk 2 above)
Comment (MBMIII) [reply src/test/java/executor/service/StepTest.java]: updated

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**